app/main.py

```diff
@@ -2,5 +2,16 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    my_cache = {}
+
+    def wrapper(*args, **kwargs) -> Callable:
```

Comment:
Zawsze warto dodawać `@wraps(func)` z modułu `functools` przed wewnętrzną funkcją aby zachować oryginalne właściwości dekorowanej funkcji, takie jak jej nazwę, dokumentację itp. 

```suggestion
    @wraps(func)
    def wrapper(*args, **kwargs) -> Callable: